changes to header, footer and service navigation (#4547)
alphagov/govuk-design-system: src/components/footer/index.md

---

{% from "_example.njk" import example %}
{% from "_brand-callout.njk" import brandCallout %}
{%- from "govuk/components/tag/macro.njk" import govukTag -%}

The GOV.UK footer provides copyright, licensing and other information about your service.

{{ example({ group: "components", item: "footer", example: "default", id: "default-1", html: true, nunjucks: true, open: false, size: "m", loading: "eager" }) }}

If you use the page template, you'll also get the footer without having to add it, as it's included by default. However, if you want to customise the default footer, read the [page template guidance about customising components](/styles/page-template/#changing-template-content).

{% call brandCallout({ text: "Brand", classes: "brand-tag-green" }) %}

> <h2>Brand changes to the {{title}} component</h2><p class="govuk-body">From June 2025, the {{title}} component will be updated to support a wider refresh of the GOV.UK brand. <a href="#/">See the refreshed GOV.UK brand identity.</a></p>

Reviewer: ```suggestion
<h2>Brand changes to the {{title}} component</h2><p class="govuk-body">From June 2025, the {{title}} component will change in support a wider refresh of the GOV.UK brand. </a></p>
```
